spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/web/ServerProperties.java

```diff
@@ -729,6 +729,11 @@ public void setCacheTtl(Duration cacheTtl) {
 		 */
 		private Integer selectors;
 
+		/**
+		 * Maximum size, in bytes, of the HTTP response header.
+		 */
+		private int maxHttpResponseHeaderSize;
```

Comment:
I don't think that matches what [our intent was](https://github.com/spring-projects/spring-boot/issues/13831#issuecomment-409617001). By customizer, we mean that a user should create a customizer if they want to customize the response size. Please remove this new property and its handling.